CONTRIBUTING.rst

```diff
@@ -152,6 +152,17 @@ these guidelines:
 -   Adhere to guidelines for commit messages described in this `article <http://chris.beams.io/posts/git-commit/>`__.
     This makes the lives of those who come after you a lot easier.
 
+Airflow Git Branches
+====================
+
+All new development in Airflow happens in ``master`` branch. All PRs should target that branch.
+We also have ``v1-10-test`` branch which is used to test ``1.10.x`` series of Airflow and where committers
+(and only committers) cherry-pick selected commits from the master branch. The ``v1-10-test`` branch might be
```

Comment:
Should we add hint only use `cherry-pick -x` from master? And should we also add this in https://github.com/apache/airflow/blob/master/.github/PULL_REQUEST_TEMPLATE.md ?

Reply:
Good point. I reworded it slightly and added -x

Reply:
I think it's ok as it is now wit PR template - the template is shown AFTER you make PR already  and master is default branch - you'd have to make a deliberate effort to change the PR target branch, and by that time you probably read about branching strategy :).